library/css/components/typography.css

```diff
@@ -59,6 +76,7 @@
 .ui.one-liner-captions {
     color: var(--caption-grey);
     line-height: 1.2rem;
+    font-size: 16px;
```

Comment:
one-liner-captions is used in other places as well, so select specifically for that card only and change there